README.md

```diff
@@ -3,3 +3,19 @@
 This repository contains metadata that is used by the nodes for automated
 bundle discovery and distribution.
 
+## List of Bundles
```

Comment:
Would it make sense to separate Mainnet and Testnet bundles?

Reply:
Yes, will do both.